fifthfile.py

```diff
@@ -0,0 +1,118 @@
+"""
+Homework.
+"""
+# Строки с заданным символом.
+# Напишите программу, которая бы работала следующим образом
+# - находила символ "#" и если этот символ найден - удаляла
+# предыдущий символ из строки. Ваша задача обработать строки с "#" символом.
+# Примеры:
+# "a#bc#d" ==>  "bd"
+# "abc#d##c"      ==>  "ac"
+# "abc##d######"  ==>  ""
+# "#######"       ==>  ""
+# ""              ==>  ""
+
+
+def delete_character(string):
+    """
+    Removes the previous character if the current character is "#".
+    """
+    result = []
+    for i in string:
+        if i == "#" and result:
+            result.pop()
+        else:
+            result.append(i)
+    return "".join(result)
+
+
+print(delete_character("a#bc#d"))
+print(delete_character("abc#d##c"))
+print(delete_character("abc##d######"))
+print(delete_character("#######"))
+print(delete_character(""))
```

Comment:
use assertion here to check that code works as expected

Reply:
ready